.github/CONTRIBUTING.md

```diff
@@ -110,23 +112,23 @@ In the following cases, you must record new HTTP requests:
 - an existing test is updated and does a new HTTP request
 - gspread is updated and does a new HTTP request
 
-### Run test, capturing *all* HTTP requests
+### Run test, capturing _all_ HTTP requests
 
 In some cases if the test suite can't record new episodes, or it can't replay them offline, you can run a complete update of the cassettes.
 
 ```bash
 GS_CREDS_FILENAME=<./YOUR_CREDS.json> GS_RECORD_MODE=all tox -e py
 ```
 
-### Run test, capturing *only new* HTTP requests
+### Run test, capturing _only new_ HTTP requests
```

Comment:
same here why change this part ?

Reply:
I don't know why that happened.

Reply:
Bo worries, probably "auto-format" in your Editor. 
Juste revert the changes and we'll be fine.